src/features/content-insights/ContentInsightsSummary.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../styles/variables';
+
+.ContentInsightsSummary {
+    display: flex;
+    flex-direction: column;
+
+    .ContentInsightsSummary-header {
+        position: relative;
+        margin: $bdl-grid-unit * 7.5 0 $bdl-grid-unit * 2.5;
```

Comment:
minor: should we use explicit margin styling rather than the shorthand to be more clear? (i.e. margin-top, margin-left, etc.)

Reply:
It's possibly but the `margin` shorthand notations are fairly commonly used so I think it should be ok this way as well